src/api/controllers/home.js

```diff
@@ -9,7 +9,7 @@ export default {
     const result = {
       status,
       data: {
-        message: 'Welcome on Authors Heaven',
+        message: 'Welcome to Authors Haven',
```

Comment:
@MCFrank16 Nice

Reply:
The change is not related to your PR but I like that you noticed the issue with the text.